src/_posts/databases/postgresql/2000-01-01-timescaledb.md

```diff
@@ -0,0 +1,280 @@
+---
+title: TimescaleDB®
+nav: TimescaleDB®
+modified_at: 2023-12-11 12:00:00
+tags: databases postgresql timescaledb timeseries extensions
+index: 11
+---
+
+
+TimescaleDB® is an open-source extension to PostgreSQL® designed to make SQL
+scalable for time series data.
+
+This extension adds several features and functions. You can find more
+information on the [official documentation](https://docs.timescale.com/api/latest).
+
+## TimescaleDB® at Scalingo
+
+{% note %}
+TimescaleDB® has some specificities which you should be aware of before getting
+started.
+{% endnote %}
```

Comment:
suggestion: I wouldn't have this sentence in a not bloc. I think that the information it contains does not requires that much highlight

```suggestion
TimescaleDB® has some particularity which you should be aware of before getting
started.
```

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/2389/commits/fc868ed5a747ffe69e0914b5fa96d9e3ab6ca37f